swim/events.go

```diff
@@ -31,7 +31,7 @@ type EventListener interface {
 	HandleEvent(events.Event)
 }
 
-// A MaxPAdjustedEvent occurs when the disseminator adjusts the max propogation
+// A MaxPAdjustedEvent occurs when the disseminator adjusts the max propagation
 // count for changes
```

Comment:
This was simply a find/replace of a misspelling. I'm not too keen on going through the entire swim codebase and adding full-stops to the end of all the sentences.